src/app/Components/States/ZeroState.tsx

```diff
@@ -18,7 +19,7 @@ export const ZeroState = (props: ZeroStateProps) => {
 
   const { title, bigTitle, subtitle, image, callToAction, minHeight = 90 } = props
 
-  if (enableRedesignedSettings) {
+  if (enableRedesignedSettings && props.showBorder) {
```

Comment:
Can we destructure `showBorder` with the other props? 

```suggestion
  if (enableRedesignedSettings && showBorder) {
```

Reply:
sure, I missed that while making the other changes